polaris-react/src/components/Filters/Filters.scss

```
@@ -1,295 +1,261 @@
+    height: auto;
+    overflow: visible;
+  }
+}
+
+.hideQueryField .FiltersWrapper {
   display: flex;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  align-items: center;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  justify-content: space-between;
 }
 
-.FiltersDesktopContainerContent {
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  width: 100%;
-  // stylelint-disable -- generated by polaris-migrator DO NOT COPY
-  height: calc(
-    100% - #{$list-filters-footer-height} - #{$list-filters-header-height}
-  );
-  // stylelint-enable
-  padding: var(--p-space-2);
+.FiltersInner {
+  overflow: auto;
+  white-space: nowrap;
+  padding: var(--p-space-3) var(--p-space-4) var(--p-space-5);
 }
 
-.FiltersMobileContainerContent {
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  width: 100%;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  height: calc(100% - #{$list-filters-header-height});
-  padding: var(--p-space-2);
+.hideQueryField .FiltersInner {
+  flex: 1;
 }
 
-.FiltersContainerFooter {
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  position: absolute;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  bottom: 0;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  width: 100%;
-  padding: var(--p-space-4) var(--p-space-5);
-  border-top: var(--p-border-divider);
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  height: $list-filters-footer-height;
-  box-sizing: border-box;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  display: flex;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  align-items: center;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  justify-content: space-between;
+// stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+@media #{$p-breakpoints-md-up} {
+  .FiltersInner {
+    overflow: visible;
+    flex-wrap: wrap;
+    gap: var(--p-space-2);
+    padding: calc(var(--p-space-2) - var(--p-space-05)) var(--p-space-2);
+  }
 }
 
-.FiltersMobileContainerFooter {
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  width: 100%;
-  padding: var(--p-space-4) var(--p-space-4);
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  height: $list-filters-footer-height;
-  box-sizing: border-box;
+.AddFilter {
+  background: var(--p-surface-subdued);
+  border-radius: $border-radius;
+  border: var(--p-border-subdued) dashed $border-width;
+  padding: calc(var(--p-space-1) - var(--p-space-05))
+    calc(var(--p-space-2) - var(--p-space-025))
+    calc(var(--p-space-1) - var(--p-space-05))
+    calc(var(--p-space-3) - var(--p-space-025));
```

```suggestion
  padding: var(--p-space-05) var(--p-space-2) var(--p-space-05) var(--p-space-3);
```

The top and bottom padding math isn't needed as `calc(var(--p-space-1) - var(--p-space-05))` ===  2px === `var(--p-space-05)`.

Could the 7px right 11px left padding values be bumped up 1px to use the spacing scale instead?